resources/views/maps/groups/create.blade.php

```diff
@@ -27,20 +27,47 @@
         @endif
         <div class="panel-body">
             @include('partials.errors')
-
-            {!! Form::open(['route' => ['maps.map_groups.store', $map], 'method' => 'POST', 'id' => 'map-group-form', 'enctype' => 'multipart/form-data', 'class' => 'ajax-subform']) !!}
-            @include('maps.groups._form', ['model' => null])
-
-            <div class="form-group">
-                <div class="submit-group">
-                    <button class="btn btn-success">{{ __('crud.save') }}</button>
-                    @includeWhen(!request()->ajax(), 'partials.or_cancel')
-                </div>
-                <div class="submit-animation" style="display: none;">
-                    <button class="btn btn-success" disabled><i class="fa-solid fa-spinner fa-spin"></i></button>
+            {!! Form::open(['route' => ['maps.map_groups.store', $map], 'method' => 'POST', 'data-shortcut' => 1]) !!}
+                @include('maps.groups._form', ['model' => null, 'map' => $map])
+                <div class="pull-right">
```

Comment:
Also missing a "cancel" button that either closes the modal, or goes back when the url is opened in a new tab. Same as we have on other modals.
![image](https://user-images.githubusercontent.com/3263440/196011169-a6a87adc-d48e-48a9-b93c-986a04553fa7.png)